### PR TITLE
Add translation to brazilian portuguese

### PR DESCRIPTION
**Pull request by Felipefpl**

### CODE REVIEW

**Review by Zangetsu38 — APPROVED** [review]
Thx i change juste some things later on my pr thx for you time

File: lang/pt_BR.xml [hunk 1]
```diff
@@ -0,0 +1,225 @@
+<?xml version="1.0" encoding="utf-8"?>
+<main_menubar>
+	<file name="File">
```
Comment (Zangetsu38) [inline lang/pt_BR.xml:3]: You have forget all atribute value like here for name

**Comment by Zangetsu38** [comment]
Also file name is wrong is pt-br look here
https://github.com/Vita3K/Vita3K/blob/e62ebc5e00c16081f1a5631409284807bdfa311a/lang/pt-br.xml

**Comment by Felipefpl** [comment]
> Thx i change juste some things later on my pr thx for you time

It's me who should thank you for the changes you did to my file and for all your work with emulator, thank you very much. :)

Btw, i noticed in the quickstart section of the site that there is no mention about why vita3k needs opengl 4. 1 and above to emulate a portable videogame that isnt as powerful as ps3, just a small hint. ;)

**Comment by Zangetsu38** [comment]
Opengl request no relate powerfull is for compatibilité, and invite you go on discord

**Pull request merged**